libsolidity/analysis/DeclarationContainer.cpp

```diff
@@ -138,19 +138,22 @@ vector<Declaration const*> DeclarationContainer::resolveName(ASTString const& _n
 vector<ASTString> DeclarationContainer::similarNames(ASTString const& _name) const
 {
 	static size_t const MAXIMUM_EDIT_DISTANCE = 2;
+	// because the best known upper bound for calculating the edit distance is quadratic, we only report similar strings if they are not too long
```

Comment:
Please change to: because the function below has quadratic runtime - it will not magically improve once a better algorithm is discovered ;)